packages/react-i18n/src/test/i18n.test.ts

```diff
@@ -229,6 +229,61 @@ describe('I18n', () => {
     });
   });
 
+  describe('#ordinal()', () => {
+    it('calls translate() utility wth ordinal scope and replacement amount', () => {
+      const defaultTranslations = [{hello: 'Hello, {name}!'}];
+      const i18n = new I18n(defaultTranslations, defaultDetails);
+
+      i18n.ordinal(0);
+      expect(translate).toHaveBeenCalledWith(
+        'other',
+        {scope: 'ordinal', replacements: {amount: 0}, pseudotranslate: false},
+        defaultTranslations,
+        i18n.locale,
+      );
+
+      i18n.ordinal(1);
+      expect(translate).toHaveBeenCalledWith(
+        'one',
+        {scope: 'ordinal', replacements: {amount: 1}, pseudotranslate: false},
+        defaultTranslations,
+        i18n.locale,
+      );
+
+      i18n.ordinal(2);
+      expect(translate).toHaveBeenCalledWith(
+        'two',
+        {scope: 'ordinal', replacements: {amount: 2}, pseudotranslate: false},
+        defaultTranslations,
+        i18n.locale,
+      );
+
+      i18n.ordinal(3);
+      expect(translate).toHaveBeenCalledWith(
+        'few',
+        {scope: 'ordinal', replacements: {amount: 3}, pseudotranslate: false},
+        defaultTranslations,
+        i18n.locale,
+      );
+
+      i18n.ordinal(4);
+      expect(translate).toHaveBeenCalledWith(
+        'other',
+        {scope: 'ordinal', replacements: {amount: 4}, pseudotranslate: false},
+        defaultTranslations,
+        i18n.locale,
+      );
+
+      i18n.ordinal(42);
+      expect(translate).toHaveBeenCalledWith(
+        'two',
+        {scope: 'ordinal', replacements: {amount: 42}, pseudotranslate: false},
+        defaultTranslations,
+        i18n.locale,
+      );
+    });
+  });
+
```

Comment:
Should there also be a test that it returns the expected values in each case, not just the interaction with `translate()`? There's nothing here that tests that it actually uses the value it gets back from `translate()`.

Reply:
Can't. [`translate()` is mocked](https://github.com/Shopify/quilt/blob/c38f6f7d4a9856a6a7880423362d92fc53b254b2/packages/react-i18n/src/test/i18n.test.ts#L12) so it just returns `undefined` unless we force a return value using `translate. mockReturnValue(....)`, but I don't see any value in that for these tests.